lib/three.js

```diff
@@ -10,7 +10,7 @@ if (THREE.TextureLoader) {
 require('../node_modules/three-dev/examples/js/loaders/OBJLoader');  // THREE.OBJLoader
 require('../node_modules/three-dev/examples/js/loaders/ColladaLoader');  // THREE.ColladaLoader
 require('../lib/vendor/Raycaster');  // THREE.Raycaster
-require('../node_modules/three-dev/examples/js/controls/VRControls');  // THREE.VRControls
-require('../node_modules/three-dev/examples/js/effects/VREffect');  // THREE.VREffect
+require('../node_modules/webvr-libs/VRControls');  // THREE.VRControls
```

Comment:
when this is merged upstream and three-dev is bumped, we'll point back to @ngokevin's